README.md

```diff
@@ -6,32 +6,48 @@
 - [Development](#dev)
 - [Contributing](#contrib)
 
-## <a name="usage"></a> Usage
+## <a name="usage"></a>Usage
 
-### Usage as a module (no source code)
+Plugin is supposed to be used to execute tests related to changes done locally on developer's machine and in CI environment to test pull requests.
 
-1. Install the module with `pip install pytest-rts`
-2. Use the tool with `pytest --rts`
+### Initialization
 
-More detailed usage is described in the [tutorial][tutorial]
+To start using pytest-rts build of coverage DB is needed. For [Trunk Based Development](https://trunkbaseddevelopment.com/) mapping database from `master` branch should be used, for [A successful Git branching model](https://nvie.com/posts/a-successful-git-branching-model/) - `develop`
 
-### Usage from source code
+1. Install [pytest-cov](https://github.com/pytest-dev/pytest-cov) with `pip install pytest-cov`
+2. Configure coverage with `.coveragerc`:
+   ```dosini
+   [run]
+   source = <path to your package>
+   relative_files = True
+   dynamic_context = test_function
+   ```
+2. Execute `pytest --cov-config=.coveragerc` which will run the entire test suite and build a mapping database
```

Comment:
Couple of things are wrong here in my opinion:

1. running `pytest --cov-config=.coveragerc` doesn't produce a coverage file, at least for me. It requires `pytest --cov --cov-config=.coveragerc`
    * But here you could just provide the path to code with the flag without needing the `source` option in the config
2. Providing `dynamic_context = test_function` results in different test function names (which are very hard to map to actual pytest test names). The pytest-cov way would be to provide the flag `--cov-context=test` without anything in the config file

Reply:
Thanks, fixed. Is it correct now?